src/ReadVTK.jl

```diff
@@ -125,7 +125,7 @@ function VTKFile(filename)
   end
 
   # Ensure correct version
-  @assert version == v"1.0"
+  @assert version ≥ v"1.0"
```

Comment:
```suggestion
  @assert version >= v"1.0"
```
We prefer to only use ASCII characters for code